code/datums/mutations.dm

```diff
@@ -294,6 +295,7 @@ GLOBAL_LIST_EMPTY(mutations_list)
 	owner.update_transform()
 	owner.pass_flags &= ~PASSTABLE
 	owner.visible_message("<span class='danger'>[owner] suddenly grows!</span>", "<span class='notice'>Everything around you seems to shrink..</span>")
+	owner.dna.species.speedmod -= 1
```

Comment:
If you ever had anything else that modified the species speedmod you could use this to reset it and dodge that penalty as well